cpu/esp_common/periph/flash.c

```diff
@@ -58,8 +58,8 @@
 /* the external pointer to the system MTD device */
 mtd_dev_t* mtd0 = 0;
 
-mtd_dev_t  _flash_dev;
-mtd_desc_t _flash_driver;
+mtd_dev_t  _flash_dev = {0};
```

Comment:
this is not necessary, default is zeroed for global vars.

Reply:
I can also make it `static` while I'm at it, this is not used outside this file.